examples/research_projects/codeparrot/README.md

```diff
@@ -49,6 +49,7 @@ The raw dataset contains many duplicates. We deduplicated and filtered the datas
 - filtering with a probability of 0.7  of files without a mention of the keywords `def` , `for`, `while`  and `class`
 - filtering files that use the assignment operator `=` less than 5 times 
 - filtering files with ratio between number of characters and number of tokens after tokenization < 1.5 (the average ratio is 3.6)
+- deduplication using minhash and jaccard similarity. minhash with a jaccard threshold 0.85 is first used to create duplicate clusters. Then these clusters are then reduced to unique files based on the exact jaccard similarity. See minhash_deduplication.deduplicate_dataset for detailed description.
```

Comment:
```suggestion
- near deduplication using Minhash and Jaccard similarity. Minhash with a Jaccard threshold 0.85 is first used to create duplicate clusters. Then these clusters are then reduced to unique files based on the exact Jaccard similarity. See `minhash_deduplication.deduplicate_dataset` for detailed description.
```

Reply:
Maybe you can move this comment up right after the line about exact deduplication